plaso/parsers/spotlight_storedb.py

```diff
@@ -896,19 +899,50 @@ def _ReadRecordPage(self, file_object, file_offset):
     page_header, bytes_read = self._ReadPropertyPageHeader(
         file_object, file_offset)
 
-    if page_header.property_table_type != 0x00000009:
+    if page_header.property_table_type not in (0x00000009, 0x00001009):
       raise errors.ParseError(
           'Unsupported property table type: 0x{0:08x}'.format(
               page_header.property_table_type))
 
     page_data = file_object.read(page_header.page_size - bytes_read)
 
+    file_offset += bytes_read
+
     if page_header.uncompressed_page_size > 0:
```

Comment:
Pleased unnest/simplify. eg:

```
if page_header.uncompressed_page_size <= 0:
  raise errors.ParseError('Unsupported compression type')

compressed_page_data = page_data
...
``

Reply:
so the data can be uncompressed as well, moving the lz4 decompression to a separate function helps improve readability.